commander/src/commander-plugin.c

```diff
@@ -796,8 +792,32 @@ plugin_cleanup (void)
   }
 }
 
-void
-plugin_help (void)
+
+static void
+plugin_commander_help (G_GNUC_UNUSED GeanyPlugin *plugin, G_GNUC_UNUSED gpointer pdat)
+{
+  utils_open_browser("https://plugins.geany.org/commander.html");
+}
+
+
+/* Load module */
+G_MODULE_EXPORT
+void geany_load_module(GeanyPlugin *plugin)
 {
-  utils_open_browser (DOCDIR "/" PLUGIN "/README");
+	/* Setup translation */
+	main_locale_init(LOCALEDIR, GETTEXT_PACKAGE);
+
+	/* Set metadata */
+	plugin->info->name = _("Commander");
+	plugin->info->description = _("Provides a command panel for quick access to actions, files and more");
+	plugin->info->version = VERSION;
+	plugin->info->author = "Colomban Wendling <ban@herbesfolles.org>";
+
+	/* Set functions */
+	plugin->funcs->init = plugin_commander_init;
+	plugin->funcs->cleanup = plugin_commander_cleanup;
+	plugin->funcs->help = plugin_commander_help;
+
+	/* Register! */
+	GEANY_PLUGIN_REGISTER(plugin, 226);
```

Comment:
style: space before open parenthesis (OK, it was wrong for `PLUGIN_VERSOIN_CHECK()` before, but well)